src/json.cpp

```diff
@@ -357,9 +357,8 @@ static void to_surrogate_pair(uint32_t unicode, uint16_t *uc, uint16_t *lc)
   *lc = (n & 0x3FF) | 0xDC00;
 }
 
-#define is_space(c) ((c) == '\t' || (c) == '\n' || (c) == '\r' || (c) == ' ')
-#define is_digit(c) ((c) >= '0' && (c) <= '9')
-
+bool is_space        (const char *c);
+bool is_digit        (const char *c);
```

Comment:
I'm working on it in https://github.com/sass/libsass/pull/1395.

There are other inconsistencies I want to fix.